.github/actions/publish-release/README.md

```diff
@@ -0,0 +1,41 @@
+# publish-release composite action
+
+This action allows you to set up a release cycle for 
+your image without worrying about the hard stuff.
```

Comment:
This synopsis is pretty vague. While the description of the `image` input is a bit more specific adequate I think your documentation wants to explain what we're starting with and what we get after we run this workflow. If we're really not worrying about the hard stuff we should document what we're accomplishing. My guess is that this action is basically "given a dockerfile (as stored in a specific special location in this repository) and the required secrets, build an image, tag it, and release it somewhere". Is there anything else the consumer needs to know?